rollup/internal/controller/watcher/chunk_proposer.go

```diff
@@ -44,22 +38,13 @@ type ChunkProposer struct {
 	proposeChunkFailureTotal           prometheus.Counter
 	proposeChunkUpdateInfoTotal        prometheus.Counter
 	proposeChunkUpdateInfoFailureTotal prometheus.Counter
-	chunkTxNum                         prometheus.Gauge
```

Comment:
Even though we don't have the limit for tx per chunk anymore might be interesting to see this in Grafana no?

Reply:
Sounds good (also thought of this, but still deleted it for the first version). Added it back for metric and db value in [34a6351](https://github.com/scroll-tech/scroll/pull/1658/commits/34a63512c01ca3f2052f310351b80dca8d1ee4c9).